### DISCUSSION TITLE
How to define the style of element DIV  in BottomToolbar

### DISCUSSION THREAD

**Discussion by nmnmtttt** [react-page/react-page#1030] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34001432/122899246-3dbbf400-d37e-11eb-91d6-ae1fb0a983c5.png)

i want to define style of the area in picture。 but i can not find the way to define them after i read the example code

**Comment by macrozone** [comment, marked as answer]
hi @nmnmtttt,

what exactly do you want to adjust?

**Reply by nmnmtttt** [reply]
the min-width ,width ,max-height.
I want the form to be suspended on the left and the height and width of the form to be adjusted

**Reply by nmnmtttt** [reply]
I can't do it by changing the style of BottomToolbar

**Reply by macrozone** [reply]
i see, i created a feature request: https://github.com/react-page/react-page/issues/1032

**Reply by macrozone** [reply]
just be aware that the current solution is more sophisticated than it seems. It tries to not overlap the whole screen while still giving as much space as needed. It adjust responsivily and also takes into account when you have multiple controls (see this example: https://react-page.github.io/examples/decorateplugins)

I worked quite some time on that. At some point i also tried out to align everything on the left, but it felt wrong to me. WE could of course also add an align configuration somewhere

**Reply by nmnmtttt** [reply]
got it，thank you
